packages/mobile/src/store/account/sagas.ts

```diff
@@ -2,11 +2,11 @@ import { accountActions, getContext } from '@audius/common'
 import accountSagas from 'common/store/account/sagas'
 import { updateProfileAsync } from 'common/store/profile/sagas'
 import { takeEvery, call } from 'typed-redux-saga'
+
+import { IS_MOBILE_USER } from 'app/constants/storage-keys'
 const { signedIn } = accountActions
 
-export const RESET_REQUIRED_KEY = 'password-reset-required'
-export const ENTROPY_KEY = 'hedgehog-entropy-key'
-export const IS_MOBILE_USER_KEY = 'is-mobile-user'
+// const RESET_REQUIRED_KEY = 'password-reset-required'
```

Comment:
delete?

Reply:
So there's a commented out section below it that used it, so I kept this around so when we do work on that, we have the key avail?